demisto_sdk/commands/common/hook_validations/readme.py

```diff
@@ -240,6 +276,44 @@ def verify_readme_image_paths(self) -> bool:
             return False
         return True
 
+    def verify_readme_relative_urls(self) -> bool:
+        """ Validate readme (not pack readme) relative urls.
+
+                Returns:
+                    bool: True If there are no invalid relative urls.
+                """
+        # If there are errors in one of the following validations return False
+        error_list = self.check_readme_relative_url_paths()
```

Comment:
why do we need this function, compared to just using `check_readme_relative_url_paths`?